build.sh

```diff
@@ -151,7 +152,7 @@ start_mwaa() {
     cp -r ../tests/integration/data/v_$version/* ./dags/data
 
     DOCKER_COMPOSE_PROJECT_NAME=aws-mwaa-local-runner-$version
-    docker-compose -p $DOCKER_COMPOSE_PROJECT_NAME -f ./docker/docker-compose-local.yml up -d --wait
```

Comment:
I removed `--wait` as the command exited on my computer with a message saying the `ontainer aws-mwaa-local-runner-2_5-local-runner-1 is unhealthy` but ended up starting anyway. I would recommend that the developers check the container boot progress with `docker ps` instead.
<img width="1022" alt="Screenshot 2024-08-08 at 00 34 15" src="https://github.com/user-attachments/assets/f61edb07-2540-4c20-a758-21ced8520d4d">